kibbeh/public/locales/id/translation.json

```
@@ -182,8 +182,8 @@
       "listening": "Menunggu tombol",
       "toggleMuteKeybind": "Atur tombol untuk membisu",
       "togglePushToTalkKeybind": "Atur tombol klik-untuk-bicara",
-      "toggleOverlayKeybind": "toggle overlay keybind",
-      "toggleDeafKeybind": "Toggle deafen keybind"
+      "toggleOverlayKeybind": "Atur tombol lapisan",
```

For the word toggle, use *Beralih* instead of *Atur* as *Beralih* has a closer meaning to toggle.
```suggestion
      "toggleOverlayKeybind": "Beralih tombol lapisan",
```

---

done, change it all to "Beralih"